File: MyWami/src/com/MyWami/model/ProfileModel.java
package com.MyWami.model;

/**
 * Created by robertlanter on 2/26/14.
 */
public class ProfileModel {
	private int identityProfileId;
	private String profileName;
	private int defaultProfileInd;
	private boolean selected;

	public int getIdentityProfileId() {
		return identityProfileId;
	}

	public void setIdentityProfileId(int identityProfileId) {
		this.identityProfileId = identityProfileId;
	}

	public String getProfileName() {
		return profileName;
	}

	public void setProfileName(String profileName) {
		this.profileName = profileName;
	}

	public int getDefaultProfileInd() {
		return defaultProfileInd;
	}

	public void setDefaultProfileInd(int defaultProfileInd) {
		this.defaultProfileInd = defaultProfileInd;
	}

	public boolean isSelected() {
		return selected;
	}

	public void setSelected(boolean selected) {
		this.selected = selected;
	}
}

File: MyWamiIOS/MyWami/MyWami/ProfileModel.swift
//
//  ProfileModel.swift
//  MyWami
//
//  Created by Robert Lanter on 4/21/15.
//  Copyright (c) 2015 Robert Lanter. All rights reserved.
//

import Foundation

public class ProfileModel {
    var identityProfileId: Int = 0
    var profileName: String = ""
    var defaultProfileInd: Int = 0
    var selected: Bool = false
    
    func setIdentityProfileId(identityProfileId: Int) {
        self.identityProfileId = identityProfileId
    }
    
    func getIdentityProfileId() -> Int {
        return self.identityProfileId
    }
    
    func setProfileName(profileName: String) {
        self.profileName = profileName
    }
    
    func getProfileName() -> String {
        return self.profileName
    }
    
    func setDefaultProfileInd(defaultProfileInd: Int) {
        self.defaultProfileInd = defaultProfileInd
    }
    
    func getDefaultProfileInd() -> Int {
        return self.defaultProfileInd
    }
    
    func setSelected(selected: Bool) {
        self.selected = selected
    }
    
    func getSelected() -> Bool {
        return self.selected
    }
    
    
    
    
}

